packages/shopping/src/controllers/user-order.controller.ts

```diff
@@ -35,13 +37,29 @@ export class UserOrderController {
       },
     },
   })
+  @authenticate('jwt')
   async createOrder(
     @param.path.string('userId') userId: string,
```

Comment:
Let's remove the `userId` argument please, we should take the id from `currentUser`.